commons/src/types/ScheduleEditor.ts

```diff
@@ -47,4 +41,20 @@ export interface Manifest extends NylasManifest {
   max_book_ahead_days: number;
   min_book_ahead_days: number;
   custom_fields: CustomField[];
+  events: any[]; // TODO
+}
+
+export interface EventDefinition {
```

Comment:
Can we use the same interface as `Event` instead?

Reply:
As in the `_Event` type? My thinking was that we should keep a distinction between "Instructions for creating an event" and "event that exists and is temporaly situated"